plugin/src/main/java/at/helpch/chatchat/user/ChatUserImpl.java

```diff
@@ -77,8 +81,6 @@ public boolean privateMessages() {
     public void privateMessages(final boolean enabled) {
         this.privateMessages = enabled;
     }
-
-    @Override
```

Comment:
why remove the override?

Reply:
whoops, I think i did that when fixing merge conflicts probably